### DISCUSSION TITLE
Prime Blocks with Tailwind

### DISCUSSION THREAD

**Discussion by stlucasgarcia** [primefaces/community#673] [PrimeVue Blocks] [answered]
Is it possible to use prime blocks with Tailwind? Are there any plans to support it?

**Comment by tugcekucukoglu** [comment, marked as answer]
For now, you can do it yourself using Tailwind instead of PrimeFlex. But we will also provide Tailwind support for blocks on our side.

**Reply by timothymarois** [reply]
This should be done sooner than later. I would purchase the prime blocks only if it had tailwind support @tugcekucukoglu

**Reply by tugcekucukoglu** [reply]
That sounds good. The team has already started working on this.

**Comment by tugcekucukoglu** [comment]
Also, I want to say PrimeCLT can be used for convert Primeflex to Tailwind: https://v4.primevue.org/primeclt/

**Reply by almirhodzic** [reply]
Can you describe the exact process of the conversion, including how to do it and in which directory to run the **prime pf2tw** command to convert PrimeFlex to Tailwind? That would be very helpful. Thank you very much!
